src/core/main.cpp

```diff
@@ -820,8 +820,7 @@ int main( int argc, char * * argv )
 		if( autoSaveEnabled &&
 			gui->mainWindow()->getSession() != MainWindow::SessionState::Limited )
 		{
-			gui->mainWindow()->runAutoSave();
-			gui->mainWindow()->autoSaveTimerStart();
+			gui->mainWindow()->autoSave();
```

Comment:
Our existing signal/slot usage is plenty for priority scheduling per something like [`RemotePlugin.cpp#L144`](https://github.com/LMMS/lmms/blob/master/src/core/RemotePlugin.cpp#L144)

```cpp
QThread foo = QThread();
foo.start( QThread::LowestPriority );
```